### ISSUE TITLE
Command problem on 'adt' when packaging IPA

### ISSUE THREAD

**Issue by EsqualiburDEV** [airsdk/Adobe-Runtime-Support#1711]
## Problem Description

When packaging "IPAs" on intellij (windows), with the new version of the SDK.
the command line does not take into account the space characters ' '.
This poses a problem for the launch of java (under windows), as well as for certain paths in the command parameters.

C:/Program Files/Java/jre1.8.0_112/bin/java.exe -Dapplication.home=E:\sdk_air\4.16.1 - AIR 33_1_1_779 -Dfile.encoding=UTF-8 -Djava.awt.headless=true -Duser.language=en -Duser.region=en -Xmx512m -jar E:\sdk_air\4.16.1 - AIR 33_1_1_779\lib\adt.jar -package -target ipa-ad-hoc -storetype PKCS12 -keystore E:\Repo_Git\mobile-cer\iphone_dev.p12 -storepass XXXX -provisioning-profile E:\Repo_Git\mobile-cer\MonProjet_Ad_Hoc.mobileprovision E:\Repo_Git\MonProjet\bin\MonProjet.ipa E:\Repo_Git\MonProjet\bin\MonProjet-BETA-app.xml -extdir E:\Repo_Git\ane -C E:\Repo_Git\MonProjet\bin MonProjet.swf -C E:\Repo_Git\MonProjet\src assets -C E:\Repo_Git\MonProjet\src install -C E:\Repo_Git\MonProjet\bin\assets\beta Assets.car

## Steps to Reproduce

- Package a project in IPA on Windows

## Known Workarounds

Launch the command line by hand, adding double quotes
Exemple :
"C:/Program Files/Java/jre1.8.0_112/bin/java.exe" -Dapplication.home="E:\sdk_air\4.16.1 - AIR 33_1_1_779" -Dfile.encoding=UTF-8 -Djava.awt.headless=true -Duser.language=en -Duser.region=en -Xmx512m -jar "E:\sdk_air\4.16.1 - AIR 33_1_1_779\lib\adt.jar" -package -target ipa-ad-hoc -storetype PKCS12 -keystore E:\Repo_Git\mobile-cer\iphone_dev.p12 -storepass XXXX -provisioning-profile E:\Repo_Git\mobile-cer\MonProjet_Ad_Hoc.mobileprovision E:\Repo_Git\MonProjet\bin\MonProjet.ipa E:\Repo_Git\MonProjet\bin\MonProjet-app.xml -extdir E:\Repo_Git\MonProjet\ane -C E:\Repo_Git\MonProjet\bin MonProjet.swf -C E:\Repo_Git\MonProjet\src assets -C E:\Repo_Git\MonProjet\src install -C E:\Repo_Git\MonProjet\bin\assets\beta Assets.car

**Comment by ajwfrost** [comment]
Hi

That looks like the command that is being launched by IntelliJ?
```C:/Program Files/Java/jre1.8.0_112/bin/java.exe -Dapplication.home=E:\sdk_air\4.16.1 - AIR 33_1_1_779 -Dfile.encoding=UTF-8 -Djava.awt.headless=true -Duser.language=en -Duser.region=en -Xmx512m -jar E:\sdk_air\4.16.1 - AIR 33_1_1_779\lib\adt.jar ...```

This is something invoking ADT, presumably IntelliJ itself, so the quotes around the paths would need to come from them...

Curious that this has changed between different SDK versions, as I can't see how the above command depends on the AIR SDK at all. Are you able to revert to a previous AIR SDK and see whether the problem disappears (and if so, what is the command-line being passed into ADT?)

thanks

**Comment by EsqualiburDEV** [comment]
Yes it is intellij which launches this command, with the comment "ADT command line".

the command-line being passed into ADT :
-package -target ipa-ad-hoc -storetype PKCS12 -keystore E:\Repo_Git\mobile-cer\iphone_dev.p12 -storepass xxxxxx -provisioning-profile E:\Repo_Git\mobile-cer\MonProjet_Ad_Hoc.mobileprovision E:\Repo_Git\MonProjet\bin\MonProjet.ipa E:\Repo_Git\MonProjet\bin\MonProjet-app.xml -extdir E:\Repo_Git\MonProjet\ane -C E:\Repo_Git\MonProjet\bin MonProjet.swf -C E:\Repo_Git\MonProjet\src assets -C E:\Repo_Git\MonProjet\src install -C E:\Repo_Git\MonProjet\bin\assets\beta Assets.car

Unable to make it work by downgrading air sdk.
Same when downgrading intellij.
